telegram_notifier/__main__.py

```diff
@@ -12,6 +12,7 @@
     arg_parser.add_argument(
         "--message", type=str, help="specifies the message to send to chat"
     )
+    arg_parser.add_argument("--no_escape", action="store_true", required=False,help="no escape the special characters (e.g. '.', '-') ")
```

Comment:
I'd prefer `"do not escape special characters (e.g. '.', '-')"` here to fit the form of the other help messages